WordPress/Classes/Utility/Logging/WPCrashLoggingProvider.swift

```diff
@@ -48,12 +48,11 @@ struct WPCrashLoggingDataProvider: CrashLoggingDataProvider {
     }
 
     var currentUser: TracksUser? {
-        let context = ContextManager.sharedInstance().mainContext
-
-        guard let account = try? WPAccount.lookupDefaultWordPressComAccount(in: context) else {
-            return nil
+        return ContextManager.shared.performQuery { context -> TracksUser? in
```

Comment:
This is a good approach, though I'll preemptively suggest that we add tests to cover this for both main thread and background thread use?